domains/cat.thedev.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Dogeyisyes",
+           "email": "dogegamesroblox@gmail.com",
+           "discord": "1128124810619273326"
+        },
+    
+        "record": {
+            "TXT": "https://github.com/dogeyisyes/2egf/blob/main/asd"
```

Comment:
TXT records alone doesn't do anything. Link leads to a 404 page. Invalid records.